pkgs/applications/editors/vim/plugins/overrides.nix

```
@@ -2776,6 +2776,20 @@ in
 
   rocks-nvim = neovimUtils.buildNeovimPlugin {
     luaAttr = luaPackages.rocks-nvim;
+
+    # luaInterpreter = config.programs.neovim.package.lua;
+
+    #       "${luaInterpreter.pkgs.luarocks_bootstrap}/bin/luarocks"
+    # should I add a luarocksConfig ?
+    # ["rocks_trees"] = {
+    #   {
+    #     ["name"] = "rocks.nvim",
+    #     ["root"] = "/home/teto/.local/share/nvim/rocks"
+    #   },
+
+    passthru.initLua = ''
+      vim.g.luarocks_binary = "${neovim-unwrapped.lua.pkgs.luarocks_bootstrap}/bin/luarocks"
```

issue: This won't work. Are you sure your test was in a pure environment?

```suggestion
      vim.g.rocks_nvim = {
        luarocks_binary = "${neovim-unwrapped.lua.pkgs.luarocks_bootstrap}/bin/luarocks",
      }
```

---

I had a luarocks in PATH for whatever reason. I've removed it to ease testing.

---

I've added the test I was using with NVIM_APPNAME to sandbox a bit more at runtime. It's a useful pattern we should extend to more (all?) tests in the neovim tests.

---

Tested it and it works now.